src/cli.ts

```diff
@@ -119,29 +90,18 @@ export class CLI {
             CLI.parameters.blobKey = CLI.commander.blobKey;
         }
 
-        if (CLI.commander.threads) {
-            let threads = parseInt(CLI.commander.threads);
-            if (threads) CLI.parameters.threads = threads;
+        if (CLI.commander.processes) {
+            let threads = parseInt(CLI.commander.processes);
+            if (threads) CLI.parameters.processes = threads;
         }
 
         if (CLI.commander.crawl) {
             CLI.parameters.crawlVersion = CLI.commander.crawl;
         }
-
-    }
-
-    /**
-     * Log CLI.parameters, sensitive parameters will be hidden.
-     */
-    public static logParms() {
-        console.log("Runtime parameters:");
-        for (let parm in CLI.parameters) {
-            if (!CLI.privateParms.has(parm)) {
-                console.log("\t" + parm + " = " + CLI.parameters[parm]);
-            } else {
-                console.log("\t" + parm + " = [HIDDEN]");
-            }
-        }
     }
-
 }
+
+export let params = function () {
+    CLI.init();
```

Comment:
`CLI.init()` will be called every time a parameter is requested. Can we do `CLI.init()` only once?